front/src/reducers/osrdconf2/common/testUtils.ts

```diff
@@ -0,0 +1,681 @@
+import {
+  OsrdConfState,
+  PointOnMap,
+  PowerRestrictionRange,
+} from 'applications/operationalStudies/consts';
+import { Allowance, Path } from 'common/api/osrdEditoastApi';
+import { omit } from 'lodash';
+import { Feature } from 'geojson';
+import { createStoreWithoutMiddleware } from 'Store';
+import { SwitchType } from 'types';
+import { defaultCommonConf } from '.';
+import { simulationConfSliceType } from '../simulationConf';
+import { stdcmConfSliceType } from '../stdcmConf';
+
+function createStore(
+  slice: simulationConfSliceType | stdcmConfSliceType,
+  initialStateExtra: Partial<OsrdConfState> = {}
+) {
+  return createStoreWithoutMiddleware({
+    [slice.name]: { ...defaultCommonConf, ...initialStateExtra },
+  });
+}
+
+function osrdCommonConfTestDataBuilder() {
+  return {
+    buildEngineeringAllowance: (): Allowance => ({
+      allowance_type: 'engineering',
```

Comment:
we can’t have that much test data hard coded in function body. 
turn that into json files or some factory functions